tests/test_all_databases.py

```
@@ -51,8 +51,8 @@ def test_schema_creation(self) -> bool:
             if self.db_type == 'sqlite':
                 database = SQLiteDatabase(self.db_config)
             elif self.db_type == 'duckdb':
-                print(f"\n✗ DuckDBは非対応です（32-bit Python環境のため）")
-                self.results['errors'].append("DuckDB not supported in 32-bit Python")
+                print(f"\n✗ DuckDBはサポート対象外です")
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)